scio-test/src/main/scala/com/spotify/scio/testing/JobTest.scala

```diff
@@ -148,16 +158,18 @@ object JobTest {
       tearDown()
     }
 
+    override def toString: String = state.toString
```

Comment:
This will print all members of `BuilderState` right? Make it more readable like `JobTest[ClassName]`?